docs/overview.md

```diff
@@ -0,0 +1,22 @@
+# ScalarDB Overview
+
+## What is ScalarDB?
```

Comment:
What type of serializability and linearizability is supported in ScalarDB is mentioned in `Manages consistency between multiple database`, but I think it might be also good to mention it along with `ACID transactions` in this first section.

Reply:
@feeblefakie Just remembered I left this. Do you think we don't need to mention "strict serializability" or something here? It might be a bit hard to mention it considering the analytical queries, but I think the strong correctness of ScalarDB for OLTP is one of the strong appeal points.